GalaxyZooTouchTable/GalaxyZooTouchTable/ViewModels/LevelerViewModel.cs

```diff
@@ -97,24 +100,24 @@ public void Reset()
 
         private void LevelUp()
         {
+            if (ClassificationLevel == MAX_LEVEL) return;
             LevelUpAnimation();
             IsOpen = true;
-            if (ClassificationLevel == MAX_LEVEL)
-            {
-                return;
-            }
             switch (ClassificationsThisSession)
             {
-                case int n when (n <= 6):
+                case int n when (n <= 5):
```

Comment:
I know that we set "One" to the default, but I wonder if there's a way to make that more explicit here (since it's the only level suspiciously missing from this piece of logic). There may not be.